Fix incorrect example in utime.rst

pcarver opened the pull request:
The parameter order in the example for ticks_diff is incorrect. If it's "too early" that means that scheduled time is greater than current time and if it's "running late" then scheduled time would be less than current time.

pcarver committed:
Fix incorrect example in utime.rst

The parameter order in the example for ticks_diff is incorrect. If it's "too early" that means that scheduled time is greater than current time and if it's "running late" then scheduled time would be less than current time.

dpgeorge commented:
Yes it looks like the order was wrong, thanks for picking that up!  But it would be good to change all (four) instances of it for consistency.

pcarver commented:
Sure, I'll take a look at where else it's wrong. I just fixed the ones that I was scratching my head over (because I was trying to follow the example and it wasn't working) so I didn't look at the rest of the file for the same mistake. I'll give it another look and update the PR.

pcarver committed:
Fix incorrect example in utime.rst

The parameter order in the example for ticks_diff is incorrect. If it's "too early" that means that scheduled time is greater than current time and if it's "running late" then scheduled time would be less than current time.

pcarver reviewed (commented):
Hopefully this shows up as an update to the pull request I previously submitted. I'm more familiar with Gerrit workflow than GitHub PRs.

dpgeorge commented:
Thanks for updating the patch!  Merged in 7d25a192206f7effa47c24a52607f00efe86e2ef
